providers/terraform-provider-csbpg/testfixtures/ssl_postgres/keygen.sh

```diff
@@ -0,0 +1,17 @@
+#openssl genrsa -out keys/ca.key  3072
```

Comment:
If these are just here for reference, maybe it would be worth adding a comment to say that